src/services/blocks/BlocksService.ts

```diff
@@ -91,6 +104,15 @@ export class BlocksService extends AbstractService {
 			eventDocs
 		);
 
+		// Check if the requested block is finalized
+		const finalized = await this.isFinalizedBlock(
+			api,
```

Comment:
Yea I am torn on this one too. Seems like it might just be a little cleaner to make into an options object as well. I am torn on this specifically though because of the fact that there is only one boolean being passed in out of the five params.

Reply:
Your call :)